react/src/containers/App.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import { Router, Route, Link, browserHistory, IndexRoute } from 'react-router';
+import BurgerIndex from '../components/BurgerIndex';
+import BurgerShow from '../components/BurgerShow';
+import ReviewIndex from '../components/ReviewIndex';
+import RestaurantIndex from '../components/RestaurantIndex';
+import RestaurantShow from '../components/RestaurantShow';
+import BurgerNew from '../components/BurgerNew';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+    }
+  }
+
+  render() {
+    return (
+      <div>
+      <h1>show</h1>
+      <Router history={browserHistory}>
+        <Route path='/'>
+          <Route path='/burgers' component={BurgerIndex} />
+          <Route path="/burgers/:id" component={BurgerShow} />
+          <Route path="/reviews" component={ReviewIndex} />
+          <Route path='/restaurants' component={RestaurantIndex} />
+          <Route path='/restaurants/:id' component={RestaurantShow} />
+          <Route path='burgers/new' component={BurgerNew} />
```

Comment:
inconsistent routes (last route), please fix. 

You also are getting a bit ahead of yourself. This is a LARGE PR. In the future. E.g. make a PR with ONLY the index page, then ONLY the show page, etc